### ISSUE TITLE
Compile issue for Android

### ISSUE THREAD

**Issue by jimzqw** [gusparis/react-native-month-year-picker#114]
ran into the below error when trying to compile Android. Does anyone know to fix that? 
```
org.gradle.api.ProjectConfigurationException: A problem occurred configuring project ':react-native-month-year-picker'
Caused by: org.gradle.api.artifacts.UnknownConfigurationException: Configuration with name 'compile' not found.
```

**Comment by bumjoo** [comment]
I have same problem.

**Comment by gusparis** [comment]
fixed on #140
